google/cloud/forseti/services/cli.py

```diff
@@ -113,16 +115,16 @@ def define_inventory_parser(parent):
     create_inventory_parser = action_subparser.add_parser(
         'create',
         help='Start a new inventory')
+    create_inventory_parser.add_argument(
```

Comment:
Is this what we want? shouldn't the first (positional) argument be the model name?

Reply:
The team decided to leave --import_as as is, so I have reverted the changes to original.